recipes/libsequence/meta.yaml

```diff
@@ -5,28 +5,28 @@ about:
 requirements:
   build:
     - gcc  
-    - zlib
+    - zlib {{ CONDA_ZLIB }}*
     - tbb
     - htslib {{CONDA_HTSLIB}}*
   run: 
     - libgcc
-    - zlib
+    - zlib {{ CONDA_ZLIB }}*
     - tbb
     - htslib {{CONDA_HTSLIB}}*
 build:
   number: 1
```

Comment:
this can be reset to 0

Reply:
Thanks.  Going to close and re-open. For some reason, the latest change isn't being seen.